src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -2,12 +2,21 @@
 
 public class Division {
 
+    //Function which checks if the divisor is equal to 0 to avoid division by zero.
+    private static void checkDivisor(int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException("Divisor cannot be zero.");
+        }
+    }
+
     public static final long division(int a, int b) {
+        checkDivisor(b);
```

Comment:
Run the checkDivisor function and send the divisor as the parameter in every function that does division